src/test/run-pass/tcp-stress.rs

```diff
@@ -9,8 +9,9 @@
 // except according to those terms.
 
 // ignore-android needs extra network permissions
-// ignore-openbsd system ulimit (Too many open files)
 // ignore-bitrig system ulimit (Too many open files)
+// ignore-netbsd system ulimit (Too many open files)
```

Comment:
Curious if we should get rid of these comments. Otherwise it should be good to go